modules/tensor_mechanics/src/userobjects/CrystalPlasticityStateVarRateComponentGSS.C

```diff
@@ -28,6 +30,8 @@ validParams<CrystalPlasticityStateVarRateComponentGSS>()
       "",
       "Name of the file containing the values of hardness evolution parameters");
   params.addParam<std::vector<Real>>("hprops", "Hardening properties");
+  MooseEnum crystal_type("FCC12 BCC12", "FCC12");
```

Comment:
Only 12 slip systems in BCC is not a number I see often; why did you make this notation choice? Will this newer method you've implemented not work for the 24 and 48 slip system combinations in BCC?

Reply:
The choice for the 12 slip systems is related to the fact that the current model (Kalidindi)  group coplanar slip systems and is hardcoded for that. This hardening model will not work when considering secondary slip system in FCC or secondary and tertiary slip systems in BCC with non-isotropic hardening. It will work if you consider isotropic hardening (i.e. hardening interaction matrix is full of ones). For that we need to add other parameters.

Reply:
Or add in logic the classes to check for which slip systems are provided in which order for the different types of slip systems. We don't need to make the design choice about slip system supply vs. check here (a separate issue would be a better choice), but the explanation of the limitation to 12 slip systems should be added in that documentation *.md file that should be part of this pr :-)